inst/rmarkdown/templates/utilitr/skeleton/_output.yml

```
@@ -17,3 +17,6 @@ utilitr::html_paged:
 #  front_cover: "resources/logo-utilitR.svg"
 #  knit:
 #    pagedown::chrome_print
+utilitr::bs4_utilitr:
+  new_session: yes
```

Je suis vraiment étonné que ça fonctionne. As-tu testé ? Logiquement, je m'attendrais à ce que ça plante car `new_session` est en dur dans l'output format

```suggestion
  new_session: yes
```

---

Oui, je l'ai testé des dizaines de fois.

---

Ah bon tu as essayé sans ? En effet, les fonctions de compilation forcent `new_session`

---

Non, je fais tourner `test_template.R` systématiquement. Ca répond à ta question?

---

Ok j'ai retiré, ça sert à rien